### DISCUSSION TITLE
Add Data Ingestion Endpoint

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3101] [Feature Requests]
Define/map the schema is of the third-party service data, to allow ingesting structured data from other services, such as Stripe, SendGrid, Salesforce, etc.

**Comment by rijkvanzanten** [comment]
This has since been implemented as the Flows webhook trigger!

**Reply by brett-james-rocketlab** [reply]
I'm not sure if this was fully possible without an extension - this is because if you make a Flows webhook, you'll need the thirdparty service to authenticate with directus before you can use it. Typical error example:

`{
	"errors": [
		{
			"message": "Token expired.",
			"extensions": {
				"code": "TOKEN_EXPIRED"
			}
		}
	]
}`

I attempted this on a flow normally, and with "all" for permissions on directus flows in the roles and permissions section.
